website/blog/2023-05-28-databend-weekly-95.md

```diff
@@ -0,0 +1,160 @@
+---
+title: "This Week in Databend #95"
+date: 2023-05-28
+slug: 2023-05-28-databend-weekly
+tags: [databend, weekly]
+description: "Stay up to date with the latest weekly developments on Databend!"
+contributors:
+  - name: ariesdevil
+  - name: BohuTANG
+  - name: dantengsky
+  - name: Dousir9
+  - name: everpcpc
+  - name: flaneur2020
+  - name: gitccl
+  - name: jun0315
+  - name: leiysky
+  - name: lichuang
+  - name: Mehrbod2002
+  - name: PsiACE
+  - name: RinChanNOWWW
+  - name: SkyFan2002
+  - name: soyeric128
+  - name: sundy-li
+  - name: TCeason
+  - name: Xuanwo
+  - name: xudong963
+  - name: youngsofun
+  - name: zhang2014
+  - name: ZhengLin-Li
+  - name: zhyass
+authors:
+  - name: PsiACE
+    url: https://github.com/psiace
+    image_url: https://github.com/psiace.png
+---
+
+[Databend](https://github.com/datafuselabs/databend) is a modern cloud data warehouse, serving your massive-scale analytics needs at low cost and complexity. Open source alternative to Snowflake. Also available in the cloud: <https://app.databend.com> .
+
+## What's On In Databend
+
+Stay connected with the latest news about Databend.
+
+### Native COS Support
+
+Databend now offers native support for COS! Compared to the S3 compatibility provided by COS, our native support has the following advantages:
```

Comment:
Databend now offers native support for COS (Cloud Object Storage)! In comparison to the S3 compatibility provided by COS, our native support offers several advantages:

- Fixed Signing Errors: We have addressed signing errors that occurred in certain boundary cases by supporting COS's native signature algorithm.

- Enhanced Functionality: In the future, we plan to provide additional features such as append and other functions that are not supported by S3 but are available in COS.

- Integrated Authentication: With COS's native support, we can seamlessly integrate with Tencent Cloud's RAM (Resource Access Management), STS (Security Token Service), and other services. This integration enables us to provide authentication capabilities without relying on static keys.